lessons/ro/chapter_1.yaml

```
@@ -13,6 +13,26 @@
 
     Odată ce vă veți familiariza cu Rust, puteți să vă declarați un **Rustaceu**. Așa
     se numesc toți oamenii care folosesc, contribuie sau sunt interesați de Rust.
+
+- title: What is Rust?
+  content_markdown: >
+    Rust este un limbaj de programare al sistemelor care este cunoscut pentru accentul pe care îl pune
+    siguranță, performanță și concurență.
+    
+    Rust este conceput pentru a oferi un control la nivel scăzut asupra resurselor sistemului
+    fără a sacrifica abstracţiile de nivel înalt.
+    
+    Ce are Rust diferit de alte limbaje de programare.
+    - proprietate și împrumut
+    - durata de viață
+    - potrivire de model
+    - concurență fără curse de date (rivalitate de date)
+    - trasaturi
+    - nu nul sau colectarea gunoiului
+    - macro-uri
+    - marfa
```

"Marfă" is a vague term and it does not express the functionality of a `crate`. Since a crate is synonymous with a ‘package’ in other programming languages and it has already been translated as "cutii" in other chapters, I think it would be useful to make the following change:
```suggestion
    - pachete numite "cutii" (crates)
```